makemessages.sh

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
+cmd="python manage.py makemessages --all --no-wrap --no-location --no-obsolete"
+echo $cmd
```

Comment:
```suggestion
echo "${cmd}"
```

Sorry, but even small and not-yet-super-important shell scripts should be clean and fault tolerant. :)